src/data/rat-partner.json

```diff
@@ -0,0 +1,89 @@
+                    "url": "/assets/img/rat-partner-option-1_en.png",
+                    "left": true,
+                    "alt": "RAT-Partner Option 1"
+                },
+                "title": "Option 1: CWA rapid test portal",
+                "text": "For rapid test partners who do not yet have their own software solution for test processing in use, we offer a <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki/CWA-Schnelltest-Portal' target='_blank' rel='noopener noreferrer' title=''>CWA rapid test portal</a>  with basic functionalities to be able to transmit your rapid test results to the CWA. The CWA rapid test portal allows you to easily connect your rapid tests into the CWA if you do not already have test management software in use. This solution has basic functionalities such as the digital recording of the proband's personal data and test results, and also automatically transmits findings to the CWA if the proband has consented to this. The prerequisite for using the web-based portal is the availability of internet-capable end devices and an internet connection at the test site. Access is via web browsers (e.g. Firefox, Internet Explorer, Microsoft Edge, Safari, etc.). Using the CWA rapid test portal, you can handle tests for all probands, no matter if it's with or without CWA. A video introduction on how to use the quick test portal can be found on <a href='https://www.youtube.com/watch?v=wAwPuVZBF3U' target='_blank' rel='noopener noreferrer' title=''>YouTube</a>. For further information, please refer to the Wiki: <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki' target='_blank' rel='noopener noreferrer' title=''>Wiki: CWA Schnelltest-Portal</a>"
+            },
+            "simple-list2": {
+                "image": {
+                    "url": "/assets/img/rat-partner-option-2_en.png",
+                    "right": true,
+                    "alt": "RAT-Partner Option 2"
+                },
+                "title": "Option 2: Integration into existing software",
+                "text": "For partners who have already developed their own software solution or use a third-party software solution, we offer an interface for connecting partner systems to the Corona-Warn-App. Prerequisites for the integration of the Corona-Warn-App are: <ul><li>Implementation of a CWA QR code (3 possible variants) in the test process</li><li>Implementation of a backend interface</li><li>Compliance with privacy notices</li></ul> For further information please refor to the <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/raw/master/docs/CWA%20Schnelltest_3rd_Party_Info_1.1.pdf' target='_blank' rel='noopener noreferrer' title=''>presentation </a> or the <a href=https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki/Anbindung-an-CWA-mit-Verwendung-von-DCCs' target='_blank' rel='noopener noreferrer' title=''>Wiki: Anbindung der Partnersysteme.</a>"
```

Comment:
refor => refer

Reply:
Will be fixed in the next commit, thanks